bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_TypeI_DiTau/WGammaFusion/baseDatacards/WGammaFusion_thirdParZZZ_secondParYYY_firstParXXX_run_card.dat

```diff
@@ -0,0 +1,265 @@
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000 = nevents ! Number of unweighted events requested 
+      0       = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+        1     = lpp1    ! beam 1 type 
+        1     = lpp2    ! beam 2 type
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'   = pdlabel   ! PDF set                                  
+ 315200 = lhaid ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
you want the LO pdf, right

Reply:
Thank you. I have removed the .swp files.
After discussion it turns out we should use a run_card as used by the same processes in https://github.com/Hooooon12/genproductions/tree/newproduction/bin/MadGraph5_aMCatNLO/cards/HeavyNeutrino_Dilepton_NLO 
I have to follow up with the authors to make sure I can run locally on them.

For what concerns the test you might have done and the message "CI: All jobs have failed", please not that in order to run locally with the current datacards I have use the branch "mg265UL".

Reply:
sorry, i don't get your point "CI: All jobs have failed", and I wonder the reason for using this specific pdf

Reply:
Thanks for the feedback and sorry it tooks sometime to reply. We followed up and made the run_card consistent with 

https://github.com/Hooooon12/genproductions/tree/newproduction/bin/MadGraph5_aMCatNLO/cards/HeavyNeutrino_Dilepton_NLO

It turned out that the syntax to get the correct systeamatics is 
$DEFAULT_PDF_SETS = lhaid 

This is now implemented in the datacards.

Kindly let us know whether there is any other follow up or you can validate the datacards
and we can proceed with the production.

Sincerely,
Francesco

Reply:
Hi one more follow up. 
If you can let us know that everything is correct. Perhaps we can converge and start the gridpack production.
As we understand, the Run2 samples have priority at the moment, but this will change soon in favor of Run3 samples,
so would be nice to have the submission ready asap.

Kindly let us know and thanks.
Francesco